tests/foreman/cli/test_host.py

```diff
@@ -1909,9 +1904,8 @@ def test_positive_attach(
         }
     )
     host = target_sat.cli.Host.info({'name': host_subscription_client.hostname})
-    host_subscription_client.register_contenthost(
-        module_org.name, activation_key=module_ak_with_cv.name
-    )
+    result = host_subscription_client.register(module_org, None, module_ak_with_cv.name, target_sat)
```

Comment:
Thanks for pointing out. I will update it.

Reply:
Running `host_subscription_client.register` then is not needed, since `host_subscription_client` already comes subscribed, see the `host_subscription_client` fixture.

Can you either
* use `rhel7_contenthost` directly and remove the fixture or
* remove all occurrences of  `host_subscription_client.register`?